daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -130,6 +131,7 @@ commandParser = subparser $ fold
         <*> (JsonApiOptions <$> many (strOption (long "json-api-option" <> metavar "JSON_API_OPTION" <> help "Pass option to HTTP JSON API")))
         <*> (ScriptOptions <$> many (strOption (long "script-option" <> metavar "SCRIPT_OPTION" <> help "Pass option to DAML script interpreter")))
         <*> stdinCloseOpt
+        <*> (SandboxClassic <$> flagYesNoAuto "sandbox-classic" False "Run with old sandbox." idm)
```

Comment:
```suggestion
        <*> (SandboxClassic <$> flagYesNoAuto "sandbox-classic" False "Run with sandbox-classic." idm)
```
I somewhat wonder if `flagYesNoAuto` is really a good idea for this. I think we originally added it because negative options sound weird (I think `--open-browser=no` was the first one) but for this one and quite a few others, I’m not sure an `auto` option makes sense and whether it needs to take an argument at all. e.g. `daml start --sandbox-classic` doesn’t have the issue of having a negative option.

Reply:
Ok, changed it to switch.